Update: provide Wiki links for new functions

SlySven opened the pull request:
Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven committed:
Update: provide Wiki links for new functions [skip ci]

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

vadi2 reviewed (approved):
You'll have to run it through CI in order to pass Github checks we've enabled.

SlySven commented:
> You'll have to run it through CI in order to pass Github checks we've enabled.

:man_facepalming: *\*sigh\**

vadi2 commented:
It makes sense, because there would be no point to the checks if you could just opt into skipping them.

Kebap commented:
> It makes sense, because there would be no point to the checks if you could just opt into skipping them.

It was possible earlier and it's exactly for cases like this, where code logic is not touched at all, at all

vadi2 commented:
That's for intermediate commits, but the final one has to be CI-approved.

SlySven committed:
Update: provide Wiki links for new functions

Make a trivial change to cause rebuild - take out unneeded `QStringLiteral`
wrappers around perfectly usable raw C strings.

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven commented:
Sorry for the delay - `git gui` was playing silly buggers with me and I could not get signing to work at all, at all. I needed to fixup up my `/usr/local/bin/pinentry` to point to `/usr/local/bin/pinentry-qt5` instead of `/usr/local/bin/pinentry-tty` (there is probably something I should have tweaked but since I was on FreeBSD - and I do not *think* that uses `alternatives` - that seemed to bodge it enough for my purposes...!)

Kebap commented:
> the final one has to be CI-approved.

I thought I remember a few instances of whole PR going in unchecked, see here for example #455 but it's 2017

vadi2 commented:
We didn't have any checks back then, see the checks tab in there.

Kebap commented:
Ok I need to find later examples then but what I recall is there deliberately were no checks listed in these PR
